templates/components/common/responsive-img.html

```diff
@@ -0,0 +1,50 @@
+{{!--
+This file implements the lazysizes srcset pattern (as seen on
+https://github.com/aFarkas/lazysizes#modern-transparent-srcset-pattern) for maximum performance on modern browsers
+and reasonable compatibility with legacy browsers.
+
+There are a few important arguments to know about when using this component:
+`lazyload`:
+    - If set to 'lazyload', the browser will not load the full-size image at first, instead the lazysizes plugin will
+        load it progressively after page load. This improves the load speed of the page, but it should be disabled for
+        high-priority above-the-fold images.
+    - If set to 'lazyload+lqip', will load a low-res image which will be replaced by lazysizes - ensuring that SOME
+        content is shown immediately on page load. Without LQIP, a transparent gif will be used instead, which reduces
+        the number of requests needed to load the page, making it even faster.
+    - If set to 'disabled', will not attempt to lazyload the image and instead tell the browser to load it immediately.
+        Use this for high-priority images that are above the fold.
+
+`fallback_size`:
+    This is the size (as specified in config.json) to use for this image in legacy browsers that do not
+    support srcset.
+
+`class`:
+    The CSS classes to add to the image, e.g. "card-image". If lazyloading is enabled, all images will have the
+    "lazyload" class so that lazysizes will target them automatically.
+
+`otherAttributes`:
+    Any other HTML attributes you want on the img tag, for example "height='100' width='100'"
+--}}
+<img {{#if image}}{{!-- Get the default image for legacy browsers that do not support srcset
+--}}src="{{getImage image (default fallback_size 'srcset_size_160')}}" alt="{{image.alt}}" title="{{image.alt}}" {{!--
+  Allow lazysizes to generate the 'sizes' attribute automatically --}}data-sizes="auto"
+  {{#unless lazyload '==' 'disabled'}}
+    {{#if lazyload '==' 'lazyload+lqip'}}
+    {{!-- If we want a LQIP, include it as the default srcset (will be overwritten by lazysizes) --}}
+    srcset="{{getImage image (default lqip_size 'srcset_size_80')}}"
```

Comment:
I notice we no longer show a loading spinner. Is it because we want to show LQIP image instead?

Reply:
Yes, either a LQIP or nothing (transparent 1px gif as inline image).

We've gotten a lot of negative feedback on the spinner.

Reply:
Yep, it makes sense. Not a big fan of the gigantic loading spinner as well. 😄 I think as long as we give _some_ indication that something is loading in the background (either a low res image or a blank space), we're good.